index.js

```diff
@@ -154,8 +154,10 @@
         enableOnClickOutside: function() {
           var fn = this.__outsideClickHandler;
           if (typeof document !== "undefined") {
-            document.addEventListener("mousedown", fn);
-            document.addEventListener("touchstart", fn);
+            var events = this.props.eventTypes || ["mousedown", "touchstart"];
```

Comment:
It might be a good idea to pull the default event types into a "constant" at the top of the file